.circleci/config.yml

```diff
@@ -2,40 +2,15 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/elixir:1.5.1
+      - image: circleci/elixir:1.6-node-browsers
```

Comment:
does this image effectively cover the `run` steps removed below?

Reply:
That's exactly right, here's the doc:
https://circleci.com/docs/2.0/circleci-images/#language-image-variants

Reply:
wow, super cool!